docs/src/overview/build-deploy.md

```diff
@@ -1,118 +1,76 @@
 ---
-title: "Build & Deploy"
+title: "Build and deploy"
 weight: 2
+description: |
+
 ---
+The build process looks through the configuration files in your repository and assembles the necessary containers on every Git push.
+The deploy process makes those containers live, replacing the previous versions, with virtually no interruption in service.
+Both processes are triggered by a push to a live branch or by activating an [environment](/administration/web/environments.md) for a branch.
+![Build pipeline](/images/workflow/build-pipeline.svg "0.50")
 
-Every time you push to a live branch (a git branch with an active environment attached to it)
-or activate an [environment](/administration/web/environments.md) for a branch,
-there are two main processes that happen: **Build** and **Deploy**.
+## Glossary
 
-1. The build process looks through the configuration files in your repository and assembles the necessary containers.  
-2. The deploy process makes those containers live, replacing the previous versions, with virtually no interruption in service.
+**Build hook**: It comprises one or more shell commands that you write to finish creating your production code base.
```

Comment:
Two things:
1. I think the best way to do a list of definitions is through description lists (formerly known as definition lists):
   ```markdown
   Build hook
   : Here is the definition
   ```
2. I think glossaries are useful for general terms that won't be covered in the article. These hooks seem very specific to the build and deploy process, so I probably wouldn't include them here, but would define them below in context. The template references "Terms ... that are not specific to the concept being explained". I'd think of it as like prerequisite terms.